example/full/instance.properties

```diff
@@ -40,7 +56,7 @@ sleeper.vpc=1234567890
 sleeper.vpc.endpoint.check=true
 
 # The subnet to deploy ECS tasks to.
```

Comment:
Does this comment need updating to "A comma separated list of subnets to deploy ECS tasks to."?